### DISCUSSION TITLE
fixme=* layer

### DISCUSSION THREAD

**Discussion by matkoniecz** [streetcomplete/StreetComplete#4119] [Q&A] [answered]
Is it feasible and a good idea to have layer showing `fixme` labels?

**Comment by kmpoppe** [comment]
Given the rather complex and unintuitive ways other mobile editors make `fixme`s available, I'd say it's at least a good idea. Unsure about feasibility.
K

**Comment by HolgerJeromin** [comment]
What is the user supposed to do with the fixme?

**Reply by matkoniecz** [reply]
Remove it as invalid, create note with photo allowing to resolve it, drop down to another editor like Vespucci, edit it with one of layers...

**Comment by kmpoppe** [comment]
Same as with a note? "Can you contribute/add something to this?" And then maybe remove the fixme, add a note with its content and the user's answer?

**Comment by westnordost** [comment]
You do have this in your fork, don't you? So you have experience on how actionable these are in general right?

**Reply by matkoniecz** [reply]
Very useful if you are an experienced editor - but about 60% requires dropping into Vespucci but leaving note with photo would be also useful, about 15% would be fixable with other layers and 25% are to be removed as not applicable.

**Reply by matkoniecz** [reply]
Though thinking about it: part of benefit comes from being notified of fixmes and fixme layer would require going into it. Maybe layers can have notification/summary listing how many nearby things are waiting for tagging?

**Reply by westnordost** [reply, marked as answer]
Doesn't sound eligible then. However, for users like you, it sounds like an implementation as a quest type would make more sense than a layer.

**Reply by mnalis** [reply]
related: https://github.com/streetcomplete/StreetComplete/issues/3084 
(also https://github.com/Helium314/StreetComplete/ fork also can show `fixme` as dots-on-map)
